test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -845,7 +845,7 @@ public void TensorFlowTransformCifar()
             var imageHeight = type.Dimensions[0];
             var imageWidth = type.Dimensions[1];
 
-            var dataFile = GetDataPath("images/images.tsv");
+            var dataFile = GetDataPath("images/imagesmixedpixelformat.tsv");
```

Comment:
>imagesmixedpixelformat [](start = 47, length = 22)

so we know, this code is not throwing exceptions, but have you looked at the output pixel to make sure the don't look scrambled? This formatting changes can seem to work on the surface, but when you look at the pixels, the might not be what you want. Especially b/c we don't have a baseline for this test.

Reply:
Yep verified, please see my response below. Great question.